### PR TITLE
Bugfix/ionicons warnings

### PR DESCRIPTION
**Pull request by ron-huberfeld**
Fix ionicons warnings

### CODE REVIEW

File: app/templates/calendar/layout.html [hunk 1]
```diff
@@ -7,7 +7,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <!-- Fonts -->
-    <link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
```
Comment (yammesicka) [inline app/templates/calendar/layout.html]: JS should come one line before the closing of the body (`</body>`)

File: app/templates/partials/base.html [hunk 2]
```diff
@@ -17,7 +17,8 @@
   <link href="{{ url_for('static', path='/global.css') }}" rel="stylesheet">
 
   <!-- Icons -->
-  <script src="https://unpkg.com/ionicons@5.2.3/dist/ionicons.js"></script>
+  <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
```
Comment (yammesicka) [inline app/templates/partials/base.html:30]: Move all the JS and audio to be before the closing of the body

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/templates/calendar/layout.html (hunk 1 above)
Comment (ron-huberfeld) [reply app/templates/calendar/layout.html]: Take a look at the warning comment.(screenshot in the issue) 
I just did what they suggested.
For some reason it is not according to standard.

Comment (ron-huberfeld) [reply app/templates/calendar/layout.html]: @yammesicka - done

File: app/templates/partials/base.html (hunk 2 above)
Comment (ron-huberfeld) [reply app/templates/partials/base.html:30]: Done

**Review by yammesicka — APPROVED** [review]

**Pull request merged**